common-functions.js

```diff
@@ -0,0 +1,125 @@
+var dateArray =[];
+var dateArray2 =[];
+
+function dateFormatSlashDDMM(a) {
+	this.dateArray=a;
+	var output1 = this.dateArray[2]+"/"+this.dateArray[1]+"/"+this.dateArray[0];
```

Comment:
use directly the input parameter name here instead of this.dateArray
e.g : date[2]